fractal_tasks_core/create_ome_zarr_multiplex.py

```diff
@@ -67,25 +68,26 @@ def create_ome_zarr_multiplex(
     :param output_path: parent folder for the output path, e.g.
                         ``"/outputpath/*.zarr"``
     :param metadata: standard fractal argument, not used in this task
-    :param channel_parameters: TBD
+    :param allowed_channels: TBD
     :param num_levels: number of resolution-pyramid levels
     :param coarsening_xy: linear coarsening factor between subsequent levels
-    :param metadata_table: TBD
+    :param metadata_table: "mrf_mlf" if a Yokogawa mrf & mlf file are in the 
+                            input_path folder. Alternatively, a dict with acquisitions
+                            as keys and full paths to a csv file containing the parsed 
+                            metadata as values
     """
+    # TODO: add option to provide a dict of metadata_table strings => 1 per acquisition
 
     # Preliminary checks on metadata_table
     if metadata_table != "mrf_mlf" and not isinstance(
-        metadata_table, pd.core.frame.DataFrame
+        metadata_table, Dict
```

Comment:
Being a dict is a good sign, but we could already check for the existence of the `acquisition` key - and fail immediately otherwise.

Reply:
I added those checks at the beginning of the task now. If it's a Dict, acquisiton keys are checked and all values must be strings that end in .csv. See https://github.com/fractal-analytics-platform/fractal-tasks-core/pull/252/commits/5b6a5b383146978d863a01a96b9fc963f8b50f97